### PR TITLE
added date and time fields.  They auto populate to current date/time

### PR DESCRIPTION
**Pull request by sbindman**
I added a date time field, and create the login/logout feature.  While data is being passed from front end to Flask, nothing is being done with the date field.  This should be implemented as the next feature.

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -82,6 +83,17 @@ def login():
 		session['username'] = username
 	
 	return redirect("/")
+
+
+
+@app.route("/logout", methods=["POST"])
+def logout():
+	"""Processes logout"""
+
```
Comment (sfalkoff) [inline controller.py]: remove extra lines

File: static/customCSS.css [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-  color:#FFF;
-  display:inline-block;
-  position:absolute;
-  top:50%;
-  width:160px;
-  z-index:100;
-  text-align:center;
-  padding:10px;
-  border-radius:3px;
+    color: white;
```
Comment (sfalkoff) [inline static/customCSS.css]: for consistency, how about we stick with two spaces for indentation (not tabs)?

File: static/customCSS.css [hunk 3]
```diff
@@ -61,6 +67,20 @@ body {
...
+    font-weight: lighter;
+}
+
+.inline {
+  display: inline;
 }
+
+#geolocate {
+     margin-left: 30%;
+}
```
Comment (sfalkoff) [inline static/customCSS.css]: I'm really digging the new row you added at the bottom. Looks great fullscreen, seems to be doing something a little funky on smaller screen size.

screen shot 2015-09-27 at 2 31 45 pm

**Comment by sfalkoff** [comment]
@sbindman this is awesome. I'm testing locally - let's chat about this tomorrow.

**Comment by sbindman** [comment]
Thank you for your comments, just seeing these now (growing pains). I like the idea of having consistent formatting so will try to stick to that going forward.

**Pull request closed without merging**